kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -75,33 +72,28 @@ def kompile(
         llvm_kompile: bool = True,
         optimization: int = 0,
     ) -> 'KEVM':
-        command = ['kompile', '--output-definition', str(definition_dir), str(main_file)]
-        if debug:
-            command += ['--debug']
-        command += ['--backend', backend.value]
-        command += ['--main-module', main_module_name] if main_module_name else []
-        command += ['--syntax-module', syntax_module_name] if syntax_module_name else []
-        command += ['--md-selector', md_selector] if md_selector else []
-        command += ['--hook-namespaces', ' '.join(KEVM.hook_namespaces())]
-        command += add_include_arg(includes)
-        if emit_json:
-            command += ['--emit-json']
-        if backend == KompileBackend.HASKELL:
-            command += ['--concrete-rules', ','.join(KEVM.concrete_rules())]
-        if backend == KompileBackend.LLVM:
-            if ccopts:
-                for ccopt in ccopts:
-                    command += ['-ccopt', ccopt]
-            if 0 < optimization and optimization <= 3:
-                command += [f'-O{optimization}']
-            if not llvm_kompile:
-                command += ['--no-llvm-kompile']
         try:
-            run_process(command, logger=_LOGGER, profile=profile)
-        except CalledProcessError as err:
-            sys.stderr.write(f'\nkompile stdout:\n{err.stdout}\n')
-            sys.stderr.write(f'\nkompile stderr:\n{err.stderr}\n')
-            sys.stderr.write(f'\nkompile returncode:\n{err.returncode}\n')
+            kompile(
+                main_file=main_file,
+                output_dir=definition_dir,
+                backend=backend,
+                emit_json=emit_json,
+                include_dirs=[include for include in includes if Path(include).exists()],
```

Comment:
This filtering is necessary as due to

```
KOMPILE_OPTS = -I $(INSTALL_INCLUDE)/kframework
```

the `make` targets add an include

```
-I /usr/lib/kevm/include/kframework
```

which does not necessarily exists.